core/src/collections.rs

```diff
@@ -133,6 +134,8 @@ impl MongoCollections {
                         MongoODBCCollectionSpecification::new(name, collection_type)
                     }).collect()
                 }).unwrap_or_else(|_| {
+                    // TODO: SQL-1281: Logging
+                    dbg_write!("Error getting collections for db {}");
```

Comment:
Is there a missing db name here?
```
 dbg_write!(format!("Error getting collections for db {val}")); // <- or whichever value was db
```

Reply:
ah, good catch! I actually didn't include it because I didn't want to log the name, but I forgot to take out the formatting directive!